api/.eslintrc.json

```diff
@@ -14,5 +14,6 @@
       "ecmaVersion": 2018
   },
   "rules": {
+    "no-console": "off"
```

Comment:
we probably should use one eslintrc for all repos

Reply:
yeah, good point!